src/pages/AppUsers.vue

```diff
@@ -420,6 +421,8 @@ export default {
                 actionCount: user.action_count,
               },
             ];
+          } else {
+            data.value = [];
```

Comment:
Should we rename this ref from `data` to `users` for clarity?

Reply:
Yeah that makes sense